ansible/roles/collectory/tasks/main.yml

```diff
@@ -9,6 +9,7 @@
   copy: src={{item}} dest=/tmp
   with_items:
     - collectory-sequences.sql
+    - collectory-schema.sql
   tags:
```

Comment:
Please add a 'collectory' tag to each task - this lets us run a cut down version of the playbook by specifying the tag